shared/video-sdk/develop/real-time-transcription/project-implementation/electron.mdx

```diff
@@ -238,14 +238,14 @@ Take the following steps to start, stop and query a Real-Time Transcription task
 
 1. **Query task status**
 
-    To notify the user of any change to the speech-to-text task you [Query the task status](#query-the-status-of-speech-to-text) using a `GET` request containing the `taskId` and `builderToken`.
+    To notify the user of any change to the real-time transcription task you [Query the task status](#query-the-status-of-real-time-transcription) using a `GET` request containing the `taskId` and `builderToken`.
 
     In `preload.js`, add the following method before `document.getElementById("join").onclick = async function ()`:
 
     ```javascript
-    document.getElementById("speechToTextQuery").onclick = async function ()
+    document.getElementById("realTimeTranscriptionQuery").onclick = async function ()
     {
-      var path = "/v1/projects/" + appID + "/rtsc/speech-to-text/tasks/" + speechToTextTaskId + "?builderToken=" + builderToken;
+      var path = "/v1/projects/" + appID + "/rtsc/speech-to-text/tasks/" + realTimeTranscriptionTaskId + "?builderToken=" + builderToken;
```

Comment:
Again used ""/rtsc/speech-to-text/tasks/"" for url.
Please check if it is fine?